docs/source/using_the_ve/virtual_ecosystem_in_use.md

```diff
@@ -19,105 +19,68 @@ language_info:
   name: python
   nbconvert_exporter: python
   pygments_lexer: ipython3
-  version: 3.11.9
+  version: 3.12.0rc3
 ---
 
-# Using the Virtual Ecosystem
+# Exploring the Virtual Ecosystem outputs
 
-The code below is a brief demonstration of the Virtual Ecosystem model in operation.
-The workflow of the model is:
+The code below provides a walkthrough of some basic plots for the input and output of
+the Virtual Ecosystem simulation.
 
-## Create the model configuration and initial data
+## Run the simulation
 
-Here we are using the example data supplied with the `virtual_ecosystem`
-package, which supplies a set of example data files and a simple model configuration
-to run a simulation. The following command line arguments set up the example data
-directory in Linux, Mac or Windows Subsystem for Linux (WSL).
+Before exploring the outputs you will need to run the simulation using the example data
+or your own input data. If you have already run the simulation and generated outtput
+data, you can skip to the
+[Initial state and input data](#initial-state-and-input-data) section below.
 
-```{code-cell} ipython3
-import pathlib
-
-import matplotlib.pyplot as plt
-import matplotlib.dates as mdates
-import numpy as np
-import xarray
-```
-
-If you have previously attempted to run this example it is probably a good idea to
-delete the existing virtual ecosystem example directory, as previously generated files
-can prevent the example simulation from running successfully. That can be done as
-follows.
+The following commands allow you to run the simulation from a Jupyter Notebook.
+However, you can also run the script from the command line following the [Getting
+Started](getting_started.md) instructions.
 
 ```{code-cell} ipython3
+---
+editable: true
+slideshow:
+  slide_type: ''
+tags: []
+---
```

Comment:
I have _not_ figured out why Jupyter sometimes chucks some of these tags in or how to filter them out. I think we can do so with the myst filters in jupytext, and we definitely should because we do not want this!